### DISCUSSION TITLE
0.6mm Brim on 0.4mm Nozzle

### DISCUSSION THREAD

**Discussion by i0i0-hh** [supermerill/SuperSlicer#3305] [Q&A] [answered]
Hi, recently, superslicer has been recommended to me so I am still new to it.

I was hoping to increase the thickness of brims to 0.6mm as it is easier to be removed. "Skirt" setting under the "Width & Flow" is locked and greyed out.

Am I looking at wrong spot? Could you point me to the right direction?

Thank you in advance.

**Comment by neophyl** [comment, marked as answer]
Skirt and brim are different.  They don't use the same extrusion.  It uses the first layer width setting, same as pretty much everything else on the first layer.

Making brims easier to remove is usually controlled through the Brim separation Gap setting.  By offsetting it very slightly you create a weaker join.  You have to have your first layer dialed in nicely to give a consistent first layer squish though.  Once you have that then you can adjust the gap.  The squish of the normal first layer will join the brim to the Object being printed with the gap determining how well.

**Reply by i0i0-hh** [reply]
Thanks for your reply. Yeah, I was told that it is not possible to adjust brim width separately, without changing entire first layer width.

Yeah, I got it dialled in pretty good if geometry is printed straight up, but when it has chamfer or overhang in geometry from the first layer, unfortunately, second layer gets welded to brim. To compensate it, I do tweak the gap slightly larger, but as you mentioned makes weaker join. 0.6mm brim was sort of universal lazy setting I do so I can just print without tweaking the gaps or realising too late needing to clean up afterwards.

It was useful trick in Cura for me, hopefully something similar could be implemented in SuperSlicer someday. 

Edit: I guess I can tweak skirts to be more like brim.
